api/repositories/app_repository.go

```diff
@@ -34,7 +34,7 @@ const (
 	Kind            string = "CFApp"
 	APIVersion      string = "workloads.cloudfoundry.org/v1alpha1"
 	TimestampFormat string = time.RFC3339
-	CFAppGUIDLabel  string = "apps.cloudfoundry.org/appGuid"
+	CFAppGUIDLabel  string = "workloads.cloudfoundry.org/app-guid"
```

Comment:
Nothing referenced this label?

Reply:
I couldn't find any other references to `appGuid` in the code. The tests use the "production" code so they still passed.

Reply:
must have just been a convenience label we added 👍